VideoPaper/WallpaperDetailView.swift

```diff
@@ -50,21 +51,49 @@ struct WallpaperDetailView<A: Asset>: View {
                                     .aspectRatio(1, contentMode: .fill)
                                     .frame(width: 50, height: 50)
                                     .clipShape(RoundedRectangle(cornerRadius: 8))
+                            } else {
+                                ZStack {
+                                    LinearGradient(
+                                        colors: [.gray.opacity(0.3), .gray.opacity(0.6)],
+                                        startPoint: .topLeading,
+                                        endPoint: .bottomTrailing
+                                    )
+                                    Image(systemName: "photo.on.rectangle.angled")
+                                        .font(.system(size: 18, weight: .semibold))
+                                        .foregroundStyle(.white.opacity(0.8))
```

Comment:
Wonder if this might look better in light mode as a dynamic primary `ShapeStyle`